src/server/controller/user/update.ts

```diff
@@ -1,26 +1,40 @@
+import { ActivityLogMessage } from '@meta/assessment'
 import { User, Users } from '@meta/user'
 
 import { BaseProtocol, DB } from '@server/db'
+import { ActivityLogRepository } from '@server/repository/public/activityLog'
 import { UserRepository } from '@server/repository/public/user'
 import { UserRoleRepository } from '@server/repository/public/userRole'
 
 export const update = async (
   props: {
     userToUpdate: User
     profilePicture?: Express.Multer.File | null
-    user?: User
+    user: User
   },
   client: BaseProtocol = DB
 ): Promise<User> => {
   const { userToUpdate, profilePicture, user } = props
 
   return client.tx(async (t) => {
-    // TODO: Add activity log entry (public schema?)
-
     if (user && Users.isAdministrator(user)) {
       await UserRoleRepository.update({ user: userToUpdate }, t)
     }
 
-    return UserRepository.update({ user: userToUpdate, profilePicture }, t)
+    const updatedUser = await UserRepository.update({ user: userToUpdate, profilePicture }, t)
+
+    await ActivityLogRepository.insertActivityLog(
+      {
+        activityLog: {
+          target: { userId: updatedUser.id, user: updatedUser.name },
```

Comment:
Would be interesting so save _what changed_, but probably out of this PR/project scope